config/routes.rb

```diff
@@ -1,81 +1,32 @@
 # frozen_string_literal: true
 
 # rubocop:disable Metrics/BlockLength
-# TODO: see https://github.com/SUSE/Portus/issues/1469
 Rails.application.routes.draw do
-  resources :errors, only: [:show]
-  resources :teams, only: %i[index show update] do
-    member do
-      get "typeahead/:query" => "teams#typeahead", :defaults => { format: "json" }
-    end
-  end
-  get "/teams/typeahead/:query" => "teams#all_with_query", :defaults => { format: "json" }
 
+	def draw(routes_module)
```

Comment:
Could you hide this inside of another file and inside of a module or class ? And, since this comes directly from Gitlab's source code, could you also write a comment stating where did this come from ? 😉